site/src/examples/list-box/Scrolling.tsx

```diff
@@ -0,0 +1,19 @@
+import { Option, ListBox } from "@salt-ds/core";
+import { ReactElement } from "react";
+import { largestCities } from "./exampleData";
```

Comment:
With example data in another folder, could add an inline comment to github source?